components/org.wso2.carbon.identity.conditional.auth.functions.elk/src/main/java/org/wso2/carbon/identity/conditional/auth/functions/elk/util/ElasticConstants.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (c) 2022, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ *
+ */
+
+package org.wso2.carbon.identity.conditional.auth.functions.elk.util;
+
+/**
+ * Set of the constants used in the cookie related functions.
+ */
+public class ElasticConstants {
+
+    public static final String TYPE_APPLICATION_JSON = "application/json";
```

Comment:
Lets take this from framework constants.

Reply:
Resolved with [f8c134b](https://github.com/wso2-extensions/identity-conditional-auth-functions/pull/98/commits/f8c134b5cf1651ab35b0f4d346b4a71ea711c50a)